src/mwoffliner.lib.ts

```diff
@@ -401,6 +404,10 @@ async function execute(argv: any) {
     logger.info('Copying Static Resource Files')
     await saveStaticFiles(config, zimCreator)
 
+    // TODO: refactor sequence, this only needed for mobile renderer
```

Comment:
this should be indeed part of the mobilerenderer

Reply:
Static files handle only in src/mwoffliner.lib.ts, not in renders.

Reply:
This is always the same kind of problem (like the CSS): code has to be factored properly. You have one place to save static files and only one, why not here. But the place to specify what are the static files to save that should be in the renderer. In addition it seems you save these static mobile files... all the time, even if they are not needed. which is not clean.

Reply:
I replaced modules handling to renderers themself. Now abstract desktop and abstract mobile have the ability to filter out necessary modules that come from Downloader. In the execution process now should be the exact modules we need.